README.md

```diff
@@ -124,28 +124,29 @@ python setup.py install
 
 ### To install using extensions enabled use the following command in apex folder:
 ```
+# if pip >= 23.1 (ref: https://pip.pypa.io/en/stable/news/#v23-1) which supports multiple `--config-settings` with the same key...
+pip install -v --no-build-isolation --config-settings "--build-option=--cpp_ext" --config-settings "--build-option=--cuda_ext" ./
+# otherwise
 python setup.py install --cpp_ext --cuda_ext
-```
-Note that using --cuda_ext flag to install Apex will also enable all the extensions supported on ROCm including "--distributed_adam", "--distributed_lamb", "--bnp", "--xentropy", "--deprecated_fused_adam", "--deprecated_fused_lamb", and "--fast_multihead_attn".
 
-### To install Apex on ROCm using ninja and without cloning the source
-```
-pip install ninja
-pip install -v --install-option="--cpp_ext" --install-option="--cuda_ext" 'git+https://github.com/ROCmSoftwarePlatform/apex.git'
```

Comment:
@pruthvistony Shouldn't the new command also work for this method without cloning the source?

Reply:
@jithunnair-amd ,
I have tried many options based on the https://pip.pypa.io/en/stable/news/#v23-1
but couldnt get any option to work. 
FYI Also on the Nvidia/apex repo, this section related to building apex with cloning is removed.